.github/workflows/annotation.yml

```diff
@@ -0,0 +1,24 @@
+# GitHub Action to annotate some of our test tool results
+name: annotation
+on: [push, pull_request]
+jobs:
+  flake8-annotation-python2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Flake8 with annotations - Python 2
+        # Using the flake8 options in the .flake8 file
+        uses: TrueBrain/actions-flake8@main
+  flake8-annotation-python3:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+      - name: Flake8 with annotations - Python 3
+        # Using the flake8 options in the .flake8 file
+        uses: TrueBrain/actions-flake8@main
```

Comment:
I'm quite happy to drop the Python 3 one as that's covered in your new workflow, I was just thinking about keeping the Python 2 one:
```suggestion
```

Reply:
Fixed!